website/data/docs-navigation.js

```diff
@@ -222,8 +225,7 @@ export default [
           },
           {
             category: 'multi-cluster',
-            name: 'Multi-Cluster Federation',
```

Comment:
did you intend to remove the name field?

Reply:
yep!

Reply:
hm yeah, seems like this should still be there, as there isn't an index file

Reply:
There is now! https://github.com/hashicorp/consul/pull/8531/files#diff-62b1a324286bcebe836618737b9e7704L1